### DISCUSSION TITLE
Git and Debug Console is missing

### DISCUSSION THREAD

**Discussion by tujeff** [coder/code-server#5207] [General]
After upgrade to 4.4.0, the git and debug console is missing. Any ideas?

<img width="1070" alt="Screen Shot 2022-05-15 at 10 18 51 PM" src="https://user-images.githubusercontent.com/67305851/169062174-a90e02d2-aa80-459d-b4e1-ce856088f03c.png">

**Comment by lg-nineoneninethreenine** [comment]
I think all you have to do to get the Git controls are to open a Git folder(new change by Microsoft on their Code OSS?). And for the debug console thing, maybe all you have to do is just right click on the circle(near the terminal) on your screenshot and toggle Debug Console. Could be as simple as that.

**Comment by tujeff** [comment]
1. I opened a Git folder, but it has `no Git` at all.
2. I right click, and there is `no debug console` to select.

Previous versions are Ok, only 4.4.0.

**Reply by lg-nineoneninethreenine** [reply]
Hmm, something I kinda expected. Will be back soon, I need to whip up a new dev env for this. Trust me, I could find a fix. Wait, have you checked the Settings UI for Git and Debug Console controls?

**Reply by tujeff** [reply]
Only upgrade from 4.3 to 4.4, did not change settings. I'm running both as a systemd.

One thing I noticed, its ok to running as a container Docker. Both 4.3 and 4.4 are OK.

**Comment by tujeff** [comment]
This is version 4.3.0, Git and Debug Console are display fine here.

<img width="1020" alt="Screen Shot 2022-05-18 at 12 32 52 PM" src="https://user-images.githubusercontent.com/67305851/169142388-e964e723-8c0b-481c-8a63-7af4896b8832.png">

Can anyone help? Thanks

**Comment by jsjoeio** [comment]
I think the Git tab is now under Output? 

<img width="1435" alt="image" src="https://user-images.githubusercontent.com/3806031/169412576-d4175206-0609-4cdd-a774-76b18d6f08fc.png">

**Comment by tujeff** [comment]
On my screen, only showed 3 tabs, PROBLEMS | OUTPUT | TERMINAL. You have 6 tabs. Is there any settings for this?  thanks!

**Reply by lg-nineoneninethreenine** [reply]
Well, yeah, I think you right-click on that element and choose what you need.

**Comment by tujeff** [comment]
That's what i have. Nothing to choose from.

<img width="1020" alt="Screen Shot 2022-05-19 at 4 01 44 PM" src="https://user-images.githubusercontent.com/67305851/169444026-ac376e2b-4273-456e-a99e-2e64e91489cd.png">

**Reply by lg-nineoneninethreenine** [reply]
I need some testing time for this, will report back.

**Reply by lg-nineoneninethreenine** [reply]
![Screenshot from 2022-05-20 12-10-49   edited ](https://user-images.githubusercontent.com/76845820/169469256-a5740bf5-b727-4349-a6cc-239edd212267.jpg)

Seems all fine to me

**Reply by tujeff** [reply]
Well, I’ve been using code server for 3 years, it is the first time see something like this. Doesn’t look fine on mine.

**Reply by jsjoeio** [reply]
Could it be a new extension you're using?

**Reply by lg-nineoneninethreenine** [reply]
Try a fresh install by removing all the data - keep a note of the settings and extensions, so that you can reinstall them on a new install.

**Comment by tujeff** [comment]
I use extension “notes” and “vscode-pdf”, but I removed them and did a fresh install. Same results. 😓

**Comment by tujeff** [comment]
<img width="1108" alt="Screen Shot 2022-05-20 at 11 46 39 AM" src="https://user-images.githubusercontent.com/67305851/169592919-265066ae-5288-458b-8ab6-0989accd512a.png">

<img width="1108" alt="Screen Shot 2022-05-20 at 11 46 19 AM" src="https://user-images.githubusercontent.com/67305851/169592923-e209bdeb-cb82-4923-9947-fc17d150798e.png">

**Reply by lg-nineoneninethreenine** [reply]
Did you do a fresh install by removing all stored data in your machine?

**Comment by tujeff** [comment]
FYI - Openvscode server 1.67.1 is fine here; and as i mentioned before code server 4.3.0 is fine too.

<img width="1200" alt="Screen Shot 2022-05-20 at 11 55 16 AM" src="https://user-images.githubusercontent.com/67305851/169594398-81d3baec-41f7-4fd1-a2c0-857bd12b023a.png">

<img width="1108" alt="Screen Shot 2022-05-20 at 11 54 44 AM" src="https://user-images.githubusercontent.com/67305851/169594428-5c6212d0-9135-4945-a098-89d12efa9d8d.png">

**Comment by tujeff** [comment]
I have debian bullseye, and use systemctl, so i can test each one of them.

<img width="1055" alt="Screen Shot 2022-05-20 at 12 00 22 PM" src="https://user-images.githubusercontent.com/67305851/169594961-5538e455-36da-4337-8ad0-72fe4630c12e.png">

thanks!
